easybuild/easyblocks/t/tensorflow.py

```diff
@@ -636,9 +640,6 @@ def configure_step(self):
                                                                              self.install_base_dir))]
             apply_regex_substitutions('configure.py', regex_subs)
 
-        # Tell Bazel to not use $HOME/.cache/bazel at all
```

Comment:
This is removed without justification and looks unrelated.

Reply:
Yes, I'm currently testing why the tests still fail (they are run at least) and found that we add to many options. Only thing needed seems to be `--output_user_root`, TEST_TMPDIR is set internally by Bazel to tell Bazel that a test is being run. Setting that for compilation is hence wrong. I'll double check if output_base and install_base can also be removed and would include that here, or shall I make a second PR where those options are removed? Problem then would be merge conflicts which I'd want to avoid.

Reply:
A second PR would be best indeed.
Perhaps a rebase can avoid merge conflicts?

Reply:
Let's see. I undid this and if this is merged first I can keep the other changes and likely rebase them